ProjectLighthouse.Servers.Website/Pages/Login/BannedUserPage.cshtml

```diff
@@ -9,6 +9,8 @@
 
     string timeZone = (string?)ViewData["TimeZone"] ?? TimeZoneInfo.Local.Id;
     TimeZoneInfo timeZoneInfo = TimeZoneInfo.FindSystemTimeZoneById(timeZone);
+
+    DateTime maximumExpiration = new(9999, 12, 31, 23, 59, 00, DateTimeKind.Utc);
```

Comment:
```suggestion
```

Reply:
This is intentional. The MaxValue expression actually exceeds the maximum value the form lets you submit.

Reply:
Is it possible to make this field static then so it isn't being recreated for every request?

Reply:
Will do that once I become available